### PR TITLE
Added Parser/DB Service API Spec

### PR DESCRIPTION
**Pull request by jaimehisao**
Putting up my API Spec up for merging after fixing comments on GIST

### CODE REVIEW

File: src/parser/apiSpec.yaml [hunk 1]
```diff
@@ -0,0 +1,126 @@
+swagger: "2.0"
+info:
+  description: "API description for the Parser/Database component of the Road Search Engine."
```
Comment (Domene99) [inline src/parser/apiSpec.yaml:3]: I don't think the description should be self aware that it's a description. I'd just remove the word "description"

File: src/parser/apiSpec.yaml [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+host: "localhost"
+basePath: "/v1"
+schemes:
+- "https"
+- "http"
+paths:
+  /parse:
+    post:
+      tags:
+      - "parser"
```
Comment (Domene99) [inline src/parser/apiSpec.yaml:17]: This doesn't match any tag. I believe it should be something like:
tags:
-name: "parser"
  description: "your description"

Comment (jaimehisao) [reply src/parser/apiSpec.yaml:17]: the `tag` is the name of the section where the `/parse` lives

File: src/parser/apiSpec.yaml [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+      keywords:
+        type: "array"
+        items:
+          type: "object"
+          $ref: '#/definitions/KeywordFrequencyPair'
+  # For Reference Only
+  KeywordFrequencyPair:
+    type: "object"
+    properties:
+      keyword:
```
Comment (Domene99) [inline src/parser/apiSpec.yaml:112]: Could you quickly update the response type please?

Comment (jaimehisao) [reply src/parser/apiSpec.yaml:112]: sure!

**Pull request closed without merging**